pubspec.yaml

```diff
@@ -5,23 +5,25 @@ version: 1.0.1
 homepage: https://github.com/Earlybyte/aad_oauth
 
 environment:
-  sdk: '>=3.0.0 <4.0.0' 
+  sdk: '>=3.3.0 <4.0.0'
+  flutter: ">=1.17.0"
 
 dependencies:
   flutter:
     sdk: flutter
   flutter_secure_storage: ^9.0.0
-  webview_flutter: ^4.4.1 
-  js: ^0.6.7 
-  http: ^1.1.0 
+  webview_flutter: ^4.4.1
+  js: ^0.6.7
+  http: ^1.1.0
   shared_preferences: ^2.2.1
   dartz: ^0.10.1
   equatable: ^2.0.5
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
+  flutter_lints: ^3.0.0
```

Comment:
we just removed a lint package 3 months ago, is there a reason for adding flutter_lints?
it was replaced with https://github.com/Earlybyte/aad_oauth/blob/master/analysis_options.yaml in https://github.com/Earlybyte/aad_oauth/pull/279/

Reply:
Ok i'll remove it